pandas/tests/io/formats/test_info.py

```diff
@@ -82,18 +78,41 @@ def test_info_categorical_column():
     df2.info(buf=buf)
 
 
-def test_info(float_frame, datetime_frame):
+def test_info_frame_float_frame_just_works(float_frame):
```

Comment:
"just_works" -> "smoke_test" maybe?

the existing test is a pattern i really dislike, where we have two unrelated fixtures for what should be separate tests (that you've separated, which i like).  but it would be nice to find a way to parametrize over float_frame/datetime_frame.  i think `indirect` might be related, but never fully got the hang of that

Reply:
I managed to do so using the approach described here: https://stackoverflow.com/a/64246323
For that, however, I needed to explicitly import ``float_frame``. Without the import it would not work.

Reply:
Now pre-commit check complains about the non-standard imports because I explicitly import fixture functions.
Please suggest

Reply:
Fixed that as @jreback suggested https://github.com/pandas-dev/pandas/pull/37887#discussion_r528248694